subsys/net/ip/tcp.c

```diff
@@ -369,15 +369,6 @@ static inline u32_t get_recv_wnd(struct net_tcp *tcp)
 	return min(NET_TCP_MAX_WIN, NET_TCP_BUF_MAX_LEN);
 }
 
-/* True if the (signed!) difference "seq1 - seq2" is positive and less
- * than 2^29.  That is, seq1 is "after" seq2.
- */
-static inline bool seq_greater(u32_t seq1, u32_t seq2)
```

Comment:
Well, it is easier to understand what the function does if it is written into function name instead of thinking some arbitrary number as a return value :)
The earlier seq_greater() is quite self explaining in this respect.